README.md

```diff
@@ -2,19 +2,50 @@
 
 An AsyncAPI schema parser for Avro 1.x schemas.
 
+> **Note**
+> Version >= `3.0.0` of this package is only supported by the `@asyncapi/parser` version >= `2.0.0`.
+
+<!-- toc is generated with GitHub Actions do not remove toc markers -->
+
+<!-- toc -->
+
+- [Installation](#installation)
+- [Usage](#usage)
+  * [Usage with remote references](#usage-with-remote-references)
+  * [Usage with local references](#usage-with-local-references)
+  * [Usage with Confluent Schema Registry](#usage-with-confluent-schema-registry)
+    + [Create an API key](#create-an-api-key)
+    + [Copy the key and the secret](#copy-the-key-and-the-secret)
+    + [Use them on your AsyncAPI document](#use-them-on-your-asyncapi-document)
+- [Features](#features)
+  * [Validation of Avro schemas](#validation-of-avro-schemas)
+  * [Support of required attributes](#support-of-required-attributes)
+  * [Support for extra attributes on top of Avro specification](#support-for-extra-attributes-on-top-of-avro-specification)
+    + [List of all supported extra attributes](#list-of-all-supported-extra-attributes)
+  * [Support for names and namespaces](#support-for-names-and-namespaces)
+- [Limitations](#limitations)
+  * [Float and double-precision numbers](#float-and-double-precision-numbers)
+  * [Hardcoded key and secret](#hardcoded-key-and-secret)
+- [Contributors](#contributors)
+
+<!-- tocstop -->
+
 ## Installation
 
-```
+```bash
 npm install @asyncapi/avro-schema-parser
+// OR
+yarn add @asyncapi/avro-schema-parser
 ```
 
 ## Usage
 
 ```js
```

Comment:
we should still keep examples with `require`
not everyone use TS and `imports` in nodejs are native in  latest versions only, afaik

Reply:
Done!